app/modules/finance/operations_router.py

```
@@ -48,13 +52,35 @@ def create_payment_voucher(payload:dict,db:Session=Depends(get_db),principal:Pri
         if not payload.get(k):raise HTTPException(400,f'{k} is required')
     if db.query(m.FinancePaymentVoucher).filter_by(school_id=principal.school_id,voucher_number=payload['voucher_number']).first():raise HTTPException(409,'Voucher number already exists')
     r=m.FinancePaymentVoucher(school_id=principal.school_id,voucher_number=payload['voucher_number'],payee=payload['payee'],amount=_money(payload['amount']),description=payload['description'],invoice_reference=payload.get('invoice_reference'),lpo_reference=payload.get('lpo_reference'),requested_by=principal.user_id);db.add(r);_audit(db,principal,'create','payment_voucher',0,f'Created voucher {r.voucher_number}');db.commit();db.refresh(r);return r
+@router.get('/finance/payment-vouchers/{voucher_id}/approval-status')
+def payment_voucher_approval_status(voucher_id:int,db:Session=Depends(get_db),principal:Principal=Depends(require_role('viewer','admin'))):
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)